.palantir/revapi.yml

```diff
@@ -1285,12 +1285,69 @@ acceptedBreaks:
     - code: "java.field.removedWithConstant"
       old: "field org.apache.iceberg.TableProperties.ROW_LINEAGE"
       justification: "Removing deprecations for 1.10.0"
+    - code: "java.method.addedToInterface"
```

Comment:
we shouldn't be breaking APIs when introducing new functionality. See also https://iceberg.apache.org/contribute/#adding-new-functionality-without-breaking-apis on how to add new stuff without breaking APIs